src/slice.ts

```diff
@@ -97,6 +103,14 @@ export function createMulticallSlice(reducerPath: string) {
           }
         })
       },
+
+      updateDefaultListenerOptions: (state, action: PayloadAction<MulticallListenerOptionsPayload>) => {
+        const { chainId, listenerOptions } = action.payload
+        const defaultListenerOptions: MulticallState['defaultListenerOptions'] = state.defaultListenerOptions
+          ? state.defaultListenerOptions
+          : (state.defaultListenerOptions = {})
```

Comment:
is this an assignment happening in the middle of the ternary? I think it should probably be unpacked

Reply:
Sounds good, done.